### ISSUE TITLE
feature request for new command :)

### ISSUE THREAD

**Issue by ludekcakl** [mbinic/vscode-tgit#9]
Hello, would it be possible to add command for **Refference Log** (/reflog)? This command is sometimes very useful for reverting back to othervise lost commits.

https://tortoisegit.org/docs/tortoisegit/tgit-dug-reflog.html

**Comment by mbinic** [comment]
Hi! It's already there, it's just named a bit differently and opens the dialog on refs/stash: (G)it Stash-L(i)st - Ctrl+G, Ctrl+I.

**Comment by ludekcakl** [comment]
Stash list is something different - it shows list of stashed changes. Reflog shows complete detailed history of all commits, merges, rebases atc..

**Comment by mbinic** [comment]
Yes and no... TorotiseGit uses the same command and dialog to display the reflog and stash list. The stash is just one more ref for it.

If you take a look here https://github.com/mbinic/vscode-tgit/blob/a77690281bdf8cbbbf9e4314837fbcc6fd13cb76/src/TGit.ts#L78, you'll see that the Stash List command is just calling the reflog command and telling it to initially show the "stash" ref (see :reflog here https://tortoisegit.org/docs/tortoisegit/tgit-automation.html#tgit-automation-basics). 

You're free to interact with any ref you want once the dialog has been opened.
